modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -219,6 +225,10 @@ function fetchFilesAndRemove(currentFilters: ?Object, size: number): Function {
       return;
     }
 
+    messageNotificationDispatcher('info',
+      <span>Removing <b>{existingFiles.length}</b> files from cart</span>,
```

Comment:
This if just removing the files in the query so it should be `size` or better yet, because I have a guard lower to make sure it doesn't try to fetch over `10000` it could say it's removing `Math.min(size, MAX_CART_SIZE)` to match.